tools/run_unit_tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright 2021-present Extreme Networks, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Run stratum unit tests inside the stratum dev container.
+# Run from /stratum.
+
+TEST_CMD="xargs -a .circleci/test-targets.txt bazel test"
+# --flaky_test_attempts to retry transient admin_service_test failures
+TEST_CMD="$TEST_CMD --flaky_test_attempts 5"
+eval $TEST_CMD
```

Comment:
I like the idea of putting "complicated" commands into scripts, but then we should also use them in CI here: https://github.com/stratum/stratum/blob/68c8bf999baf256af23f530e36cec060b9919d49/.circleci/config.yml#L95